chatwoot/chatwoot
# Fix: backend changes for custom attribute

# Pull Request Template

## Description

To remove the with_indifferent_access

Fixes  4548 

## Type of change

Please delete options that are not relevant.


## How Has This Been Tested?

On local:
 - create a new automation rule from scratch
 - check the custom attribute changes affecting any of the previous rules and actions
 
With test cases
- update automation test cases without indifferent access and test the impact.

## Checklist:
